src/components/PublicMessage.module.css

```diff
@@ -0,0 +1,7 @@
+.route-abbreviation {
+  background-color: black;
+  color: white;
+  font-size: 1.25rem;
+  font-weight: 300;
```

Comment:
I dunno, it looks ideal to me (pic attached in case it's rendering differently). Anyways I ripped these rules out of the bootstrap `.lead` class, so there is a larger consensus than just me behind it

![Screenshot 2023-08-15 at 1 17 14 PM](https://github.com/umts/public-message-board/assets/8195530/2708bcd3-1d20-4c61-a527-0b530ba62689)

Reply:
Also, not your problem, but #FFFFFF on #FF0000 fails WCAG contrast ratios unless it's "large text"